packages/elements/src/photon-multirx-form/components/AddPrescriptionCard.tsx

```diff
@@ -212,30 +213,60 @@ export const AddPrescriptionCard = (props: {
             });
           }}
         >
-          <photon-treatment-select
-            label="Search Treatment Catalog"
-            selected={props.store.treatment?.value ?? undefined}
-            invalid={props.store.treatment?.error ?? false}
-            help-text={props.store.treatment?.error}
-            off-catalog-option={offCatalog()}
-            on:photon-treatment-selected={(e: any) => {
-              if (e.detail.data.__typename === 'PrescriptionTemplate') {
-                repopulateForm(props.actions, {
-                  ...e.detail.data,
-                  notes: [e.detail.data?.notes, props.prefillNotes].filter((x) => x).join('\n\n')
-                });
-              } else {
+          {props.enableNewMedicationSearch ? (
```

Comment:
didnt we delete a bunch of code? What medication search is available if this is false?

Reply:
regular old-school `photon-treatment-select`, your friendly neighborhood medication selector